website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "GraphQL"
+id: sl-graphql
+description: "Integrate and use the GraphQL API to query your metrics."
+tags: ["semantic-layer, apis"]
+---
+
+<VersionBlock lastVersion="1.5">
+
+import UpgradeSL from '/snippets/_upgrade-new-sl.md';
+
+<UpgradeSL />
+
+</VersionBlock>
+
+
+
+
+GraphQL is an open-source query language for APIs (Application Programming Interfaces). It provides a more efficient and flexible alternative to traditional RESTful APIs. 
+
+With GraphQL, clients can request specific data requirements using a single query. GraphQL allows clients to retrieve all the needed data in a single request. This reduces the number of round trips to the server, minimizing network overhead and improving performance.
+
+Overall, GraphQL offers benefits such as being self-documenting, a strong typing system, versioning and evolution support, enabling rapid development, and a robust ecosystem. These advantages make GraphQL a powerful choice for building APIs that prioritize flexibility, performance, and developer productivity.
+
+
+### dbt Semantic Layer GraphQL API
+
+The dbt Semantic Layer GraphQL API has the following capabilities and has similarities to the calls available in our JDBC API.
+
+One benefit of GraphQL is its self documenting nature, so you can explore using our schema explorer(todolink). To give you an idea of the types of commands you can execute, we provide some available calls below.
+
+
+**Metric Metadata Calls**
```

Comment:
@rpourzand do we need to add any info on endpoints? I'm just looking at the content in the [discovery api docs](https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-querying#run-queries-using-http-requests), which tells users the request endpoints, and more.